scanner/cmd/updater/main.go

```diff
@@ -44,9 +46,21 @@ func main() {
 			if err != nil {
 				return err
 			}
+			manualURL, err := cmd.Flags().GetString("manual-url")
+			if err != nil {
+				return err
+			}
 			const retries = 3
+			if manualURL == "" {
```

Comment:
Nitpick: This won't be empty because you specified a default in the flag definition at line 82, right?